src/components/EventOverview/EventsPage.tsx

```diff
@@ -1,20 +1,45 @@
 import React, { useState } from 'react';
 import { useQuery, useMutation } from 'react-apollo';
-import { Grid, Typography } from '@material-ui/core';
-import { useHistory, useLocation } from 'react-router-dom';
+import { makeStyles, Button, Box, Grid, Typography } from '@material-ui/core';
+import { useHistory, useLocation, NavLink } from 'react-router-dom';
+import AddIcon from '@material-ui/icons/Add';
 import MenuTabs from '../common/MenuTabs';
-import AddEventButton from './AddEventButton';
 import EventCard from './EventCard';
 import UserProfile from './UserProfile';
 import useAllEvents from '../../graphql/queries/hooks/events';
 import { Event, GET_ALL_EVENTS } from '../../graphql/queries/events';
 import { EDIT_EVENT, DELETE_EVENT } from '../../graphql/mutations/events';
-import '../../styles/HomeLandingPage.css';
+import { Colours } from '../../styles/Constants';
+
+const useStyles = makeStyles({
+  root: {
+    backgroundColor: Colours.BackgroundGray,
+    minHeight: '100vh',
```

Comment:
can this be coverted to `px`?

Reply:
This should really be in `vh` since we want the minimum height of the div to take up the entire height of the screen. It's not something that can be converted to `px` necessarily.